src/c-writer.cc

```diff
@@ -635,6 +638,19 @@ void CWriter::PopLabel() {
   label_stack_.pop_back();
 }
 
+// static
+constexpr bool CWriter::HasNonNullInitializers(Type type) {
```

Comment:
Since this is always using in the negative how about inverting it?  i.e. "NullOnlyInitializers"?